core/body/body.ctrl.js

```diff
@@ -174,7 +185,42 @@ export class BodyCtrl extends View {
 		}
 	}
 
+	doBatch(startCell) {
```

Comment:
make a separate service under this core/edit/edit.service
and do `new EditService(model, table....).batchUpdate(...`